### DISCUSSION TITLE
ΓΕ2 - Ορισμός Ομάδων

### DISCUSSION THREAD

**Discussion by std159312** [sdy60/syllabus#109] [Q&A] [answered]
Καλησπέρα σας,
οι ομάδες (3-4 ατόμων) σε ποιο σημείο θα πρέπει να τις δηλώσουμε?
Ευχαριστώ

**Comment by epidrome** [comment, marked as answer]
Στην αρχή της αντίστοιχης απάντησης στο Μέρος Β στην αναφορά σας

**Comment by std133524** [comment]
Καλημέρα σας. 

Είχε αναφερθεί στην ΟΣΣ πως κάποιοι συνάδελφοι έχουν ήδη ομάδες απο τις προηγούμενες θεματικές.

Μπορούμε σε αυτό το thread να δημιουργήσουμε τις ομάδες που δεν υπάρχουν ή να τις αναφέρουμε ώστε να αρχίσουμε την μεταξύ μας συνεργασία; 

Ευχαριστώ

**Reply by iparthenios** [reply]
Ομάδα από προηγούμενη θεματική:
ΗΛΙΑΣ ΠΑΡΘΕΝΙΟΣ
ΟΔΥΣΣΕΑΣ ΖΑΓΟΡΑΣ
ΑΠΟΣΤΟΛΟΣ ΑΠΟΣΤΟΛΙΔΗΣ
Εάν επιθυμούν οι συνάδελφοι μπορούμε να συνεχίσουμε ή ακόμα και να γίνουν προσθήκες. Όλοι ευπρόσδεκτοι!

**Reply by apapost** [reply]
οκ!

**Reply by ghost** [reply]
Ναι φυσικά

**Comment by Maurphal** [comment]
Έχουμε και εμεις μια ομάδα από πέρσι
ΧΡΗΣΤΟΥ ΓΕΩΡΓΙΟΣ
ΔΟΥΒΑΛΕΤΑΣ ΝΙΚΟΛΑΟΣ
ΚΑΛΟΓΕΙΤΟΝΑΣ ΓΕΩΡΓΙΟΣ
ΔΗΛΑΒΕΡΗΣ ΑΛΕΞΑΝΔΡΟΣ
αν χρειαστεί κάποιος κάτι είμαστε διαθέσιμοι.

**Comment by std133524** [comment]
Ok, εγώ δεν είμουν σε κάποια προηγούμενη ομάδα επομένως δεν γνωρίζω αν υπάρχει χώρος . Από το study.eap.gr βλέπω πως υπάρχει μια λίστα με 12 άτομα σε αυτή την θεματική + ο καθηγητής. Επομένως σκεφτόμουν ίσως, αν συμφωνείτε και εσείς, να το κρατήσουμε σε 4 ομάδες; (12/3 =4). Δηλ 3 άτομα σε κάθε ομάδα. Αν υπάρχουν παραπάνω άτομα σε κάθε ομάδα, δεν πειράζει. Απλά να αρχίσουμε μια συζήτηση. Ευχαριστώ

**Comment by std159312** [comment]
Η  δική μας ομάδα είναι:

1. Αλτανόπουλος Ιωάννης
2. Θεοδωρόπουλος Ιωάννης
3. Σβεντζούρης Κωνσταντίνος
4. Σινάπης Άγγελος

**Comment by std133524** [comment]
Οκ. ευχαριστώ πολύ!  Από την στιγμή που είμαστε 12 και στην πρώτη ομάδα είναι 3 και στις επόμενες 2 ομάδες από 4 άτομα (@Maurphal, @std159312 ), θα πάω στην πρώτη ομάδα με τον @iparthenios για να είμαστε 3 ομάδες με 4 άτομα ακριβώς.

**Reply by iparthenios** [reply]
Νίκο αν έχεις δει τη δημιουργία team και του repo για την ομάδα θα ήθελα τη γνώμη σου.
